content/blog/automating-operations-on-dscc-using-ansible-playbooks.md

```diff
@@ -0,0 +1,437 @@
+
+3. Check the version of Ansible by using the following command:
+
+   ```shell
+   $ansible --version
+   ```
+
+For other operating systems, please refer to the official Ansible documentation.
+
+**Installing the HPE GreenLake Data Services Python SDK**
+
+These Ansible playbooks use the Python libraries of the HPE GreenLake Python SDK. Install the SDK using this command:
+
+```shell
+$pip install git@github.hpe.com:sijeesh-kattumunda/greenlake_data_services_python.git
+```
+
+Cloning the DSCC GitHub repo	
+To clone the repo, execute the following command on the machine where you installed Ansible:
+
+```shell
+$git clone git@github.hpe.com:sijeesh-kattumunda/greenlake-data-services-ansible.git
+```
+
+This repo mainly consists of two folders:
+
+1. **Libraries**: These are Python libraries that will be used by the Ansible playbooks to perform CRUD operations on DSCC resources.
+2. **Examples**: This folder consists of sample Ansible playbooks that perform CRUD operations on DSCC resources. With these examples, one can start building use cases.
+
+**Setting up environment variables**
+
+ANSIBLE_LIBRARY and ANSIBLE_MODULE_UTILS are environment variables that need to be set.
+Set the ANISBLE_LIBRARY variable to the greenlake-data-services-ansible/library directory of the repo like this.
+
+`ANSIBLE_LIBRARY=/home/admin/greenlake-data-services-ansible/library`
+
+Set the ANSIBLE_MODULE_UTILS to the module_utils directory under greenlake-data-services-ansible/library directory like this.
+
+`ANSIBLE_MODULE_UTILS=/home/admin/greenlake-data-services-ansible/library/module_utils`
+
+**Usage**
+
+Let’s have a look at an example used to perform DSCC operations on a host using an Ansible playbook. For any Ansible playbook to be used, an inventory file is required. The Ansible inventory file defines the hosts and groups of hosts on which commands, modules, and tasks in a playbook operate. In this example, we are calling REST APIs from our local machine. This file can be placed anywhere and the path of this file can be given during the Ansible playbook execution. Create an inventory file, name it “hosts” (the name of the file can be anything), and update it with the following details: 
+`[localhost]
+127.0.0.1 ansible_connection=local ansible_python_interpreter=/home/admin/ansible-env/bin/python`
+
+
+The IP address given is localhost, which means operations are performed on the local host. Provide the Python interpreter location on your system. Name this file “hosts”.
+
+Let's look at a sample of the host file.
+Ansible playbooks contain a list/array of tasks that will be performed sequentially. The below code snippet has the following tasks.
+
+1. **Creation** of host
+
+   In this block, the input request parameters are provided under the section ‘data’. In this sample, only required fields for the REST API call are provided such as name, initiator_ids, user_created flag, and operating system.
```

Comment:
escape the special character "_".

Reply:
Please escape the "underscore" character for the variables "initiator_ids" and "user_created"